### PR TITLE
Patches fetched from local mercurial

### PR DESCRIPTION
**Pull request by anubhabsen**
Issue #1506 
I think this applies the patches from the local mercurial repository using a fetch. Please do let me know if there is something I am missing, and how I should go about testing it.

### CODE REVIEW

File: scripts/commit_classifier.py [hunk 1]
```diff
@@ -355,28 +355,18 @@ def load_user(phid):
...
-                        f"user.name={author_name}",
-                        "-c",
-                        f"user.email={author_email}",
-                        "commit",
-                        "-am",
-                        message,
+                        "cinnabar",
+                        "fetch",
+                        f"hg::{self.git_repo_dir}",
+                        f"{revision['id']}",
```
Comment (marco-c) [inline scripts/commit_classifier.py:364]: This is a Phabricator revision ID, it's not the hash of the commit we just applied to the local mercurial repository.

Comment (anubhabsen) [reply scripts/commit_classifier.py:364]: Sorry about that. If I understand correctly, ```needed_stack``` contains the list of patches to be applied. So ```path.base_revision``` where patch is an element of needed_stack is the hash of a commit that needs to be applied?

Comment (marco-c) [reply scripts/commit_classifier.py:364]: No, base_revision is the revision in the Mercurial repository on top of which the patch must be applied (https://github.com/mozilla/bugbug/blob/78e5b5a961f1985d7510a74de4c5741111a58d79/scripts/commit_classifier.py#L278-L282).

The process should be:
1) Get patches from Phabricator;
2) Apply each patch to the local Mercurial repository at the specified base revision;
3) Fetch the commits from the local Mercurial repository in the local git repository.

For 3, the commits come from the local Mercurial repository, Phabricator is already out of the picture.

**Comment by marco-c** [comment]
Closing for lack of updates, feel free to reopen if you still intend to work on this.

**Pull request closed without merging**